book/src/fork-selection.md

```diff
@@ -0,0 +1,147 @@
+# Fork Selection
+This chapter defines Solana's *Nakomoto Fork Selection* algorithm based on time
```

Comment:
I don't see how this relates to Nakomoto *Block* Selection. Can you cite the description you're looking at?

Reply:
https://www.reddit.com/r/btc/comments/4r0mfa/longest_chain_or_most_work/

nodes prefer the chain with the most work, and not the "longest" one.  our chain selection rule is to pick the chain with the most lockout, and not the longest one.

Reply:
Seems I wasn't specific enough with my request. Can you cite the description in the document?

Reply:
just leaving this as a note:

Honest generators only build onto a block (by referencing it in blocks they create) if it is the latest block in the longest valid chain. "Length" is calculated as total combined difficulty of that chain, not number of blocks, though this distinction is only important in the context of a few potential attacks. A chain is valid if all of the blocks and transactions within it are valid, and only if it starts with the genesis block. (source https://en.bitcoin.it/wiki/Block_chain)